rust/private/repository_utils.bzl

```diff
@@ -205,15 +205,15 @@ load("@rules_rust//rust:toolchain.bzl", "rust_toolchain")
 
 rust_toolchain(
     name = "{toolchain_name}",
-    rust_doc = "@{workspace_name}//:rustdoc",
-    rust_std = "@{workspace_name}//:rust_std-{target_triple}",
-    rustc = "@{workspace_name}//:rustc",
+    rust_doc = "//:rustdoc",
```

Comment:
If I try to use the repository name in this, I get the following error:

```
ERROR: /home/cameron/.cache/bazel/_bazel_cameron/538574de729809a828a75cd1284ac81c/external/rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools/BUILD.bazel:89:15: no such package '@[unknown repo 'rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools' requested from @rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools]//': The repository '@[unknown repo 'rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools' requested from @rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools]' could not be resolved: Repository '@rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools' is not visible from repository '@rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools' and referenced by '@rules_rust~override~toolchains~rust_linux_x86_64__x86_64-unknown-linux-gnu_tools//:rust_toolchain'
```

Reply:
If you want to use the canonical repo name in a label string, you'll need to use `@@canonical_repo_name//foo:bar`, see https://github.com/bazelbuild/bazel/issues/15593

But I guess it's fine to use the label without repo name here since it's referring to targets in the same repo as the BUILD file resides?

Reply:
/cc @Wyverald

Reply:
Omitting the repo name looks fine to me in this context.